code/modules/mining/machine_vending.dm

```diff
@@ -289,7 +289,7 @@
 	. = ..()
 	if(istype(AM, /obj/item/card/id) && proximity)
 		var/obj/item/card/id/I = AM
-		I.access |= list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM, ACCESS_CARGO)
+		I.access |= list(ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM, ACCESS_CARGO, ACCESS_MAILSORTING)
```

Comment:
`minimal_access = list(ACCESS_MINING, ACCESS_MINT, ACCESS_MINING_STATION, ACCESS_MAILSORTING, ACCESS_MAINT_TUNNELS, ACCESS_MINERAL_STOREROOM)` is the access for miners. `minimal_access` being the access given on normal pop.
Best to just change it to this.
The `ACCESS_CARGO` access is for the shuttlebay oddly enough.

Reply:
Good idea, it is done.